pages/desktop/consumer_pages/search.py

```diff
@@ -33,7 +32,6 @@ def __init__(self, testsetup, app_name=False):
             self.app_name = app_name
         else:
             self._page_title = 'Search Results | Firefox Marketplace'
-        self.wait_for_element_present(*self._search_results_section_locator)
```

Comment:
This seems a bit dangerous to have removed a wait without finding something to replace it.